packages/Molecules/RelatedLinks/index.vue

```diff
@@ -7,7 +7,7 @@
     <div class="rpl-related-links__row">
       <ul class="rpl-related-links__items" v-if="links">
         <li class="rpl-related-links__item" v-for="(item, index) of links" :key="index">
-          <rpl-text-link :url="item.url" :text="item.text" :underline="true" size="small" theme="dark" />
+          <rpl-text-link :url="item.url" icon-symbol="arrow_right_primary" icon-color="white" icon-size="small" :text="item.text" :underline="true" size="small" theme="dark" />
```

Comment:
Hey @dylankelly The `icon-size` options are here: https://github.com/dpc-sdp/ripple/blob/master/packages/Atoms/Icon/Icon.vue#L38
```
sizes: {
        's': 0.5,
        'm': 1,
        'l': 1.5,
        'xl': 2,
        'xxl': 2.5
      }
```
Can `s`, `m`... or any number, so value `small` won't work here I think.